pkg/analysis/builtins.go

```diff
@@ -115,21 +115,13 @@ func LoadBuiltinsFromSource(ctx context.Context, contents []byte, path string) (
 		return nil, errors.Wrapf(err, "failed to parse %q", path)
 	}
 
-	functions := make(map[string]protocol.SignatureInformation)
 	doc := document.NewDocument(uri.File(path), contents, tree)
 	docFunctions := doc.Functions()
 	symbols := doc.Symbols()
 	doc.Close()
 
-	for fn, sig := range docFunctions {
-		if _, ok := functions[fn]; ok {
-			return nil, fmt.Errorf("duplicate function %q found in %q", fn, path)
-		}
-		functions[fn] = sig
-	}
-
```

Comment:
This must have been a remnant of some earlier logic. No need to dupe-check a Go map.